lib/proxy_manager.js

```diff
@@ -160,20 +160,34 @@ function loadConfigFiles(cb) {
     stop_me[proxy_config_file] = proxy;
   });
 
+  // TODO (@theopak): Consider rewriting this to make it recursive.
   fs.readdir(CONFIG_DIR, function(err, files) {
     /* istanbul ignore if */
     if (err) return cb(err);
 
+    // Require a '.json' or '.xml' file ending and reject dotfiles
+    // TODO (@theopak): Write unit tests for this.
+    var configFiles = [];
```

Comment:
This is a bit peevish, but I use snake_case for local variables in this project.